devon4j-architecture/index.asciidoc

```diff
@@ -0,0 +1,50 @@
+= Tutorial for the devon4j architecture
+====
+This tutorial aims to explain architecture for devon4j application.There are many different views that are summarized by the term architecture. It focuses on
+* *Key Principles* 
+* *Architecture Principles* 
+* *Application Architecture* 
+    * *Business Architecture*
+    * *Technical Architecture*
+====
+====
+This step explains Key principles and architecture principles.
+[step]
+--
+nextKatacodaStep("Key and Architecture Principles", [{ "file": "principles.md" }])
+--
+====
+====
+This step explains application architecture.
+[step]
+--
+nextKatacodaStep("Application Architecture", [{ "file": "architecture.md" }])
+--
+====
+====
+Now, after having detailed understanding of devon4j architecture principle and concept , in this step we will create sample devon4j application.
+Before creating devon4j application we need to install devonfw-ide.
+[step]
+--
+restoreDevonfwIde(["java", "mvn"])
+--
+Before proceeding to next steps ide should be installed.
+====
+====
+In this, we will create sample devon4j application with name sampleapp. 
+[step]
+--
+createDevon4jProject("sampleapp")
```

Comment:
I think you are creating the project to show the user the architecture, but this is missing. You should point the user to the parts of the project where the things you are mentioned are visible.